docs/jwst/mrs_imatch/LICENSE.rst

```diff
@@ -0,0 +1,5 @@
+*******
+LICENSE
+*******
+
+.. include:: ../../../LICENSE
```

Comment:
Have we adopted a convention of licensing files? Is there a need to put a License file in every step?
Normally I would expect to see one License file in the root directory. Then the first line of each file is the license information. I think we shuold keep these things as standard as possible.